tests/util.py

```diff
@@ -6,7 +6,21 @@
 
 def create_test_user(slack_id):
```

Comment:
Perhaps, we can be more explicit here and rename the method `create_test_admin_user` since admin user is not the best representation of all users. What do you think?

Reply:
Yes